Production build fails for Blog Starter Kit with Sanity template from examples.

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

The project is bootstrapped from the official templates website of Vercel at https://vercel.com/templates/next.js/blog-next-sanity.

### Which example does this report relate to?

cms-sanity

### What browser are you using? (if relevant)

_No response_

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

Next.js build fails. Here is the full stacktrace from Vercel:

```
...
[22:34:50.681] Previous build cache not available
[22:34:51.187] Cloning completed: 751.919ms
[22:34:51.633] Running "vercel build"
[22:34:52.126] Vercel CLI 28.4.17
[22:34:52.419] Installing dependencies...
[22:34:52.751] yarn install v1.22.17
[22:34:52.774] info No lockfile found.
[22:34:52.782] [1/4] Resolving packages...
[22:34:59.367] [2/4] Fetching packages...
[22:35:11.412] [3/4] Linking dependencies...
[22:35:17.105] [4/4] Building fresh packages...
[22:35:17.117] success Saved lockfile.
[22:35:17.123] Done in 24.38s.
[22:35:17.152] Detected Next.js version: 13.0.2
[22:35:17.154] Running "yarn run build"
[22:35:17.417] yarn run v1.22.17
[22:35:17.444] $ next build
[22:35:17.903] Attention: Next.js now collects completely anonymous telemetry regarding usage.
[22:35:17.903] This information is used to shape Next.js' roadmap and prioritize features.
[22:35:17.904] You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
[22:35:17.904] https://nextjs.org/telemetry
[22:35:17.904] 
[22:35:18.014] info  - Linting and checking validity of types...
[22:35:18.209] info  - Creating an optimized production build...
[22:35:22.550] Failed to compile.
[22:35:22.550] 
[22:35:22.550] ./components/avatar.js
[22:35:22.551] Module not found: Can't resolve 'next/future/image' in '/vercel/path0/components'
[22:35:22.551] 
[22:35:22.551] Import trace for requested module:
[22:35:22.551] ./components/avatar.js
[22:35:22.551] ./components/hero-post.js
[22:35:22.551] 
[22:35:22.551] ./components/cover-image.js
[22:35:22.551] Module not found: Can't resolve 'next/future/image' in '/vercel/path0/components'
[22:35:22.551] 
[22:35:22.552] Import trace for requested module:
[22:35:22.552] ./components/cover-image.js
[22:35:22.552] ./components/hero-post.js
[22:35:22.552] 
[22:35:22.552] 
[22:35:22.552] > Build failed because of webpack errors
[22:35:22.576] error Command failed with exit code 1.
[22:35:22.577] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[22:35:22.596] Error: Command "yarn run build" exited with 1
```

### Expected Behavior

I expected the build to not fail.

### To Reproduce

1. Visit https://vercel.com/templates/next.js/blog-next-sanity
2. Click "Deploy".
3. Connect Sanity project.
4. Wait for the build output.

--- a/examples/cms-sanity/components/avatar.js
+++ b/examples/cms-sanity/components/avatar.js
@@ -1,4 +1,4 @@
-import Image from 'next/future/image'
+import Image from 'next/image'
 import { urlForImage } from '../lib/sanity'
 
 export default function Avatar({ name, picture }) {

--- a/examples/cms-sanity/components/cover-image.js
+++ b/examples/cms-sanity/components/cover-image.js
@@ -1,5 +1,5 @@
 import cn from 'classnames'
-import Image from 'next/future/image'
+import Image from 'next/image'
 import Link from 'next/link'
 import { urlForImage } from '../lib/sanity'
 

--- a/examples/with-sfcc/components/Header.js
+++ b/examples/with-sfcc/components/Header.js
@@ -1,4 +1,4 @@
-import Image from 'next/future/image'
+import Image from 'next/image'
 
 export default function Header({ scrollHandler }) {
   return (

--- a/examples/with-sfcc/components/ProductCard.js
+++ b/examples/with-sfcc/components/ProductCard.js
@@ -1,4 +1,4 @@
-import Image from 'next/future/image'
+import Image from 'next/image'
 import Link from 'next/link'
 import { useState } from 'react'
 

--- a/examples/with-sfcc/pages/products/[slug].js
+++ b/examples/with-sfcc/pages/products/[slug].js
@@ -1,4 +1,4 @@
-import Image from 'next/future/image'
+import Image from 'next/image'
 import getProducts from '../../sfcc.js'
 
 export default function Product({ product }) {

